scripts/export.sh

```diff
@@ -13,76 +19,89 @@ if [[ -z "${SNAPSHOT_FILE}" ]]; then
+echo "Walking from epoch ${FROM_EPOCH} to ${TO_EPOCH}..."
+sleep 5
 
-# # Copy the exported data to the export directory
-cp -r /tmp/data/mainnet/ "${EXPORT_DIR}"
+# Run the walk job, if TASKS is provided, use it
+if [[ -z "${TASKS:-}" ]]; then
+  lily job run --storage=CSV walk --from "${FROM_EPOCH}" --to "${TO_EPOCH}"
+else
+  lily job run --storage=CSV --tasks "${TASKS}" walk --from "${FROM_EPOCH}" --to "${TO_EPOCH}"
+fi
 
-# # Alternatively, we could run the export with lily
-# lily job run --storage=CSV walk --from "${FROM_EPOCH}" --to "${TO_EPOCH}"
-# lily job wait --id 1
+# Wait for the job to finish
+echo "Waiting for job to finish..."
+lily job wait --id 1
 
-# # Fail if job error is not empty string
-# JOB_ERROR=$(lily job list | jq ".[0]" | jq -r ".Error")
-# if [[ "${JOB_ERROR}" != "" ]]; then
-#   echo "Job failed with error: ${JOB_ERROR}"
-#   cat lily.log
-#   exit 1
-# fi
+# Fail if job error is not empty string
+JOB_ERROR=$(lily job list | jq ".[0]" | jq -r ".Error")
+if [[ "${JOB_ERROR}" != "" ]]; then
+  echo "Job failed with error: ${JOB_ERROR}"
+  cat lily.log
+  exit 1
+fi
 
-# lily stop
+lily stop
 
-# # Check there are no errors on visor_processing_reports.csv
-# if grep -q "ERROR" /tmp/data/*visor_processing_reports.csv; then
-#   echo "Errors found on visor_processing_reports!"
-#   cat /tmp/data/*visor_processing_reports.csv | grep "ERROR"
-#   exit 1
-# fi
+# Check there are no errors on visor_processing_reports.csv
+if grep -q "ERROR" /tmp/data/*visor_processing_reports.csv; then
+  echo "Errors found on visor_processing_reports!"
+  grep "ERROR" /tmp/data/*visor_processing_reports.csv
+fi
 
-# # Check the chain_consensus file has WALK_EPOCHS + 1 (header) lines
+# Check the chain_consensus file has WALK_EPOCHS + 1 (header) lines
 # if [[ $(wc -l < /tmp/data/*chain_consensus.csv) -ne $((WALK_EPOCHS + 1)) ]]; then
 #   echo "chain_consensus file has $(wc -l < /tmp/data/*chain_consensus.csv) lines, expected $((WALK_EPOCHS + 2))"
 #   exit 1
 # fi
 
-# # Compress the CSV files
-# gzip /tmp/data/*.csv
+# Compress the CSV files
+echo "Compressing CSV files..."
+gzip /tmp/data/*.csv
+
+# Move files to export dir
+echo "Saving CSV files to ${EXPORT_DIR}"
+
+# Clean export dir
+# TODO: FILENAME should be the snapshot epoch range
+FILENAME=$(basename "${SNAPSHOT_FILE}" .car.zst)
+if [ -d "${EXPORT_DIR:?}"/"${FILENAME:?}"/ ]; then
+  rm -Rf "${EXPORT_DIR:?}"/"${FILENAME:?}"/;
+fi
+mkdir -p "$EXPORT_DIR"/"$FILENAME"/
 
-# # Move files to export dir
-# echo "Saving CSV files to ${EXPORT_DIR}"
-# FILENAME=$(basename "${SNAPSHOT_FILE}" .car.zst)
-# if [ -d "${EXPORT_DIR:?}"/"${FILENAME:?}"/ ]; then rm -Rf "${EXPORT_DIR:?}"/"${FILENAME:?}"/; fi
-# mkdir -p "$EXPORT_DIR"/"$FILENAME"/
-# mv /tmp/data/*.csv.gz "$EXPORT_DIR"/"$FILENAME"/
-# mv lily.log "$EXPORT_DIR"/"$FILENAME"/
+# Add data and log files to export dir
+mv /tmp/data/*.csv.gz "$EXPORT_DIR"/"$FILENAME"/
+mv lily.log "$EXPORT_DIR"/"$FILENAME"/
```

Comment:
This makes the CSV results of the export available in one folder. Processing `archival_snapshot_abc` will produce CSVs in `gs://.../archival_snapshot_abc`.